go.mod

```diff
@@ -11,32 +11,31 @@ require (
 	github.com/patrickmn/go-cache v2.1.1-0.20180815053127-5633e0862627+incompatible
 	github.com/pborman/uuid v1.2.1
 	github.com/sirupsen/logrus v1.9.0
-	github.com/stretchr/testify v1.8.0
-	golang.org/x/crypto v0.0.0-20220826181053-bd7e27e6170d
+	github.com/stretchr/testify v1.8.1
+	golang.org/x/crypto v0.8.0
 	gopkg.in/macaroon.v2 v2.1.0
-	gorm.io/driver/postgres v1.3.9
-	gorm.io/gorm v1.23.8
+	gorm.io/driver/postgres v1.5.2
+	gorm.io/gorm v1.25.0
```

Comment:
This change is a bit of a jump because v1.23.8 was released Sept 2022. There were two function definition changes that I needed to update our code to (the Golang compiler caught them.)

https://github.com/go-gorm/gorm/tags

A little hard to guarantee full correctness but I'm putting some of my faith in the compiler + the test coverage to ensure everything is working correctly 😬

Reply:
got it! I also did some looking and didn't see anything immediately concerning in terms of compatibility/changes.